source/SIL.AppBuilder.Portal.Frontend/src/ui/components/product-artifact/index.tsx

```diff
@@ -17,17 +17,17 @@ export default function Builds({ product }) {
     productBuilds: (q) => q.findRelatedRecords(product, 'productBuilds'),
   });
 
-  const [activeVersion, setActiveVersion] = useState((productBuilds || [])[0]);
-
-  const sortedBuilds = productBuilds.sort(compareVia((build) => attributesFor(build).version));
+  const sortedBuilds = productBuilds.sort(
+    compareVia((build) => attributesFor(build).version, true)
+  );
+  const [activeVersion, setActiveVersion] = useState((sortedBuilds || [])[0]);
 
   return (
     <div data-test-build>
       <ResourceSelect
         items={sortedBuilds}
         labelField={(build: ProductBuildResource) => {
-          console.log(build);
-          const version = attributesFor(build).version;
+          const version = attributesFor(build).version || ` ${t('projects.buildPending')} `;
```

Comment:
is the whitespace before and after `${t('projects.buildPending')}` intentional?

Reply:
it is yeah, the rendered output either be:

`Current Build (v1.0.0)`
or
`Current Build ( Build Pending )`

without the spaces it does:
`Current Build (build Pending)`
so, I guess, I should add "this" comment as an inline-comment, and then make an issue to investigate further (doing that now)